From 4fddab8c9e22a8c0f7e82a890ee9c23e511e5501 Mon Sep 17 00:00:00 2001
From: "Alexander F. Lent" <lx@xanderlent.com>
Date: Wed, 14 Aug 2024 14:49:42 -0400
Subject: [PATCH 2/2] Make firmware install respect CMAKE_INSTALL_PATH

(Upstream tried to fix this by using an absolute path, but then the
CMake install step will always try to install to /lib rather than
wherever we are pointing the install.)

Signed-off-by: Alexander F. Lent <lx@xanderlent.com>
---
 firmware/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/firmware/CMakeLists.txt b/firmware/CMakeLists.txt
index 09bf0aa..b201813 100644
--- a/firmware/CMakeLists.txt
+++ b/firmware/CMakeLists.txt
@@ -12,7 +12,7 @@ target_include_directories(${PROJECT_NAME} INTERFACE include)
 file(GLOB FIRMWARE_BINARIES ${CMAKE_CURRENT_SOURCE_DIR}/bin/*.bin)
 if (FIRMWARE_BINARIES)
   install(FILES ${FIRMWARE_BINARIES}
-    DESTINATION /lib/firmware/updates/intel/vpu/
+    DESTINATION lib/firmware/updates/intel/vpu/
     PERMISSIONS OWNER_READ
     COMPONENT fw-npu)
 endif()
-- 
2.47.0

